### ISSUE TITLE
Parse snapshots from FTP

### ISSUE THREAD

**Issue by vladvelici** [HackPartners/darwinpush#4]
Same as #3, but deal with the daily snapshots instead of the 5-min interval logs.

**Comment by axsaucedo** [comment]
This one would be of more importance than #3, as these are not just data, but metadata such as cancelationReason, all schedules for the day, etc

**Comment by vladvelici** [comment]
I agree #3 is more important, but they should have the same kind of data.

If you do a `grep -i cancelReason pPort*` on the FTP you'll see the logs have `cancelReason` as well.
